bundles/org.openhab.binding.openwebnet/src/main/java/org/openhab/binding/openwebnet/internal/handler/OpenWebNetAuxiliaryHandler.java

```diff
@@ -0,0 +1,131 @@
+import org.openwebnet4j.message.Where;
+import org.openwebnet4j.message.WhereAuxiliary;
+import org.openwebnet4j.message.Who;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link OpenWebNetAuxiliaryHandler} is responsible for sending Auxiliary (AUX) commands/messages to the bus
+ * It extends the abstract {@link OpenWebNetThingHandler}.
+ *
+ * NOTICE: Support for handling messages from the bus regarding alarm control has to be implemented
+ *
+ * @author Giovanni Fabiani - Initial contribution
+ *
+ */
+@NonNullByDefault
+public class OpenWebNetAuxiliaryHandler extends OpenWebNetThingHandler {
+
+    public static final Set<ThingTypeUID> SUPPORTED_THING_TYPES = OpenWebNetBindingConstants.AUX_SUPPORTED_THING_TYPES;
+    private final Logger logger = LoggerFactory.getLogger(OpenWebNetAuxiliaryHandler.class);
+
+    public OpenWebNetAuxiliaryHandler(Thing thing) {
+        super(thing);
+    }
+
+    /**
+     * Handles Auxiliary command for a channel
+     *
+     * @param channel the channel
+     * @param command the Command
+     */
+    @Override
+    protected void handleChannelCommand(ChannelUID channel, Command command) {
+        logger.debug("handleAuxiliaryCommand() (command={} - channel={})", command, channel);
+        Where w = deviceWhere;
+        if (w != null) {
+            if (channel.getId().equals(CHANNEL_AUX)) {
+                if (command instanceof StringType) {
+                    try {
+                        if (command.toString().equals(Auxiliary.WhatAuxiliary.ON.name())) {
+                            send(Auxiliary.requestTurnOn(w.value()));
+                        } else if (command.toString().equals(Auxiliary.WhatAuxiliary.OFF.name())) {
+                            send(Auxiliary.requestTurnOff(w.value()));
+                        }
+                    } catch (OWNException e) {
+                        logger.debug("Exception while processing command {}: {}", command, e.getMessage());
+                    }
+                } else {
+                    logger.debug("Unsupported command {} for channel {}", command, channel);
+                }
+            } else {
+                logger.debug("Unsupported ChannelUID {}", channel);
+            }
+        }
+    }
+
+    @Override
+    protected void requestChannelState(ChannelUID channel) {
+        /*
+         * NOTICE: It is not possible to get the state of a
+         * WHO=9 command. To get state of the Alarm system use WHO=5 instead
+         */
+
+        super.requestChannelState(channel);
+        Where w = deviceWhere;
+        if (w != null) {
+            try {
+                OpenMessage msg = BaseOpenMessage.parse("*#9##");
+                // initializing
+                send(msg);
+            } catch (MalformedFrameException | UnsupportedFrameException | OWNException e) {
+                logger.warn("Exception while processing command {}: ", e.getMessage());
```

Comment:
```
logger.warn("Exception while processing command: {}", e.getMessage());
```

Reply:
I would channge it to debug....

Reply:
Unfortunately, you misunderstood my comment. What I was mentioning is that the `:` was not at the right place in your log message.

Reply:
May I remove the colon?  ':'

Reply:
> logger.debug("Exception` while processing command: {}", e.getMessage()); 
i
s that ok?

Reply:
`logger.debug("Exception while processing command {}:", e.getMessage());`

Reply:
No !
`:` should before `{}` in your message.
Look at my initial comment.

Reply:
Correct message: `Exception while processing command: {}`

Reply:
Ok, debug or warn method?

Reply:
Look at in the binding what log level is generally used when catching `OWNException` for the method `send`, and use the same level.

Reply:
ok fine